docs/build-protocol-info.md

```diff
@@ -0,0 +1,168 @@
+---
+id: build-protocol-info
+title: Polkadot Protocol Information
+sidebar_label: Polkadot Protocol
+---
+
+This page serves as a high-level introduction to the Polkadot protocol with terminology that may be
+specific to Polkadot, notable differences to other chains that you may have worked with, and
+practical information for dealing with the chain.
+
+## DOT Tokens
+
+- **Token decimals:** 12
+- **Balance type:** `u128`
+
+## Addresses
+
+In Polkadot (and most Substrate chains), user accounts are identified by a 32-byte (256-bit)
+`AccountId`. This is simply the public key for the x25519 cryptography used by Substrate.
+
+Polkadot (and Substrate) use the SS58 address format. This is a broad "meta-format" designed to
+handle many different cryptographies and chains. It has much in common with Bitcoin's Check58
+format such as a version prefix, a hash-based checksum suffix, and base-58 encoding.
+
+See the [SS58 page](https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)) in
+the Substrate wiki for encoding information.
+
+### Cryptography
+
+Polkadot supports the following cryptographic key pairs and signing algorithms:
```

Comment:
Would also link to here 

https://wiki.polkadot.network/docs/en/learn-cryptography